.github/workflows/smoke-test-daily.yml

```diff
@@ -242,29 +216,18 @@ jobs:
               run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js upload-plugin.spec.js
 
             - name: Run the rest of E2E tests
-              id: e2e
               working-directory: plugins/woocommerce
               env:
                   E2E_MAX_FAILURES: 15
-              run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js basic.spec.js
```

Comment:
I added a separate config file for the meantime in order to skip the shopper tests here on the daily tests without affecting the PR smoke tests.